docs/ENV.md

```diff
@@ -54,6 +54,19 @@ These variables are specifically for a single language or tool.
 | -------- | --------------------------------------- |
 | CLJ_CMD  | Set to override the clojure cli command |
 
+### Cocoa
+
+| Variable                                | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| COCOA_EXCLUDED_TARGETS                  | Comma-separated list of targets to exclude from resolution of dependencies. If a target has nested targets, these are also excluded. For excluding only those nested targets, use a `/` to describe the target, eg `target/subtarget/subsubtarget` -- this excludes only `subsubtarget` from the resolution.                                                                                                                                                                                                                         |
+| COCOA_FULL_SCAN                         | Whether or not to do a full (deep) scan of the pods. This requires CocoaPods to be installed and runnable from the `PATH`. When set to `false` or `0`, only the most basic of information will be gathered (name, version, purl and, if applicable, sub-spec) -- can be useful if run on Windows. Defaults to `true`.                                                                                                                                                                                                                |
```

Comment:
We have options.deep. Can we use that instead of this env variable? options. deep defaults to false though.

Reply:
Depends on what `options.deep` does/means, I guess. I'll check it out.

Reply:
Looks like `options.deep` also gets set depending on profiles. We could use it, but I think we should have the ability to specifically (de)activate the full scanning. The reason being that most users will probably run cocoapods only on a Mac and a scan on any other platform might then cause problems. Then again, they might also only generate SBOMs on a Mac, where it would be a non-issue...

Reply:
Can cocoapods run with our container images once we add the missing `gem install`? We can always do `options.deep || isMac` to decide whether to do a full scan?

Reply:
It probably can, although I can't test this right now -- I currently have no docker/podman running on this system. It works in the repotests however, so if they are similar it should not be a problem. We could even add it on the linux images, if ruby is installed. Windows might be a bit more complicated, I was unable to get cocoapods running correctly, but according to my research, it should be doable.